README.md

```diff
@@ -5,17 +5,35 @@ We will be using cutting edge technology and software to engineer a fully-functi
 
 ## Installation
 
-Dependencies are intended to be installed using `conda`:
-
-```
+Software dependencies are intended to be installed using `conda`:
+```bash
 conda env create -f environment.yml
 ```
 
 Then, install the `clean_air` package to this environment using `pip`:
-
-```
+```bash
 conda activate cap_env
 pip install .
 ```
 
 Remember to use the `-e` option to `pip install` for development work.
+
+## Credentials
+Some code accesses resources held in an AWS S3 compatible object store.
+Credentials must be provided by 
+(TODO... we use boto3, so probably need the credentials stored in a way that boto3 will access)
+
```

Comment:
Is this TODO the reason for all the test fails (credentials aren't there yet, so of course they don't exist in the thing)?

If so, then I would be happy to merge this PR regardless of fails.

Reply:
That was more that I wasn't sure how credentials needed to be stored. I could probably put a bit about `boto3` in here, but I'm not sure how the `s3fs` based stuff gets its credentials. It might be the same as it's using `botocore`, but I hadn't looked into it to be sure